torch/_inductor/ir.py

```diff
@@ -249,7 +250,11 @@ def get_stride_order(seq: Sequence[Union[int, torch.SymInt, Expr]]) -> Sequence[
     """
     Convert strides to stride order
     """
-    sorted_idx: List[int] = argsort(seq)
+    seq_lst = list(seq)
+    if all(isinstance(elt, (int, torch.SymInt)) for elt in seq_lst):
+        sorted_idx = argsort_symint(seq_lst)
+    else:
+        sorted_idx = argsort(seq_lst)
```

Comment:
I don't know how to combine the Symint path and the Sympy.Expr path because (1) I'm not sure how to evaluate Sympy.Expr and (2) I need to guard_size_oblivious on the SymInts

Reply:
It is kind of annoying to combine the two paths because we would need to pass around V.graph (the GraphLowering?) and this needs to be threaded through a bunch of APIs

Reply:
Sorry, refreshing my memory on guard_size_obivious.. Isn't this equivalent to sizevars.statically_known_leq / other apis ? 

Can you convert the symints to sympy expr and then use those apis ?

Reply:
converting the symints to sympy expr requires me to thread through sizevars and that's a bit of a refactor... but I can attempt that if we think it's worth it? Otherwise this PR as-is fixes all my problems

Reply:
Isn't it just `s.node.expr`

Reply:
hmmm, let me try this